src/index.ts

```diff
@@ -90,6 +90,28 @@ export function getRegistrationData({
   })
 }
 
+export function getVerifyPasscode({
+  passcode,
+  programGUID, reliantGUID
+}: GetVerifyPasscodeParamType) {
+  return CompassLibraryReactNativeWrapper.getVerifyPasscode({
+    passcode,
+    programGUID, reliantGUID
+  })
+}
+
+export interface GetVerifyPasscodeParamType {
+  passcode: string,
+  programGUID: string,
+  reliantGUID: string
+}
+
+export interface GetVerifyPasscodeResultType {
+  status: boolean,
+  rId: string,
```

Comment:
The properties in the interfaces should be separated using semicolons. Please replace the commas with semicolons.

Reply:
Noted. Fixed [Here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/13/commits/bb7f6408f50ee86a8a3f882ff9f4f0bcb4b2bb51)